### PR TITLE
OOP school library: refactor your code

### PR DESCRIPTION
**Pull request by Zelalem1222**
### In this project 

-  We refactor the previous code by creating a separate files.

       - Created book_logic.rb
       - created person_logic.rb
       - created rentals_logic.rb
       - created data_values.rb

-  We make sure YAGNI, KISS, and DRY rules and Ruby style guide are respected.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,170 +1,4 @@
...
-  end
-end
-
-
-# def main
-
-# end
-
-app = Main.new
+app = App.new
```
Comment (tufoinnkuo10) [inline main.rb]: - [ ] Great job so far, refactoring the code. It works well. According to the guidelines, the main.rb should not only be responsible for starting, but should also ask the user for the initial choice of options (or to quit). Please add that to it

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  attr_reader :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def options
+    puts 'Please choose an option by enterin a number :'
```
Comment (tufoinnkuo10) [inline app.rb]: - [ optional] a welcome text can be added

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Zelalem1222 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ Descriptive pull request message
✅ well refactored
✅ Documented your work professionally

## Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Zelalem1222, and @NabilHY👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**